dietpi/dietpi-config

```diff
@@ -2082,31 +2084,32 @@
 
 				;;
 
-				"ARM Initial Turbo"*)
+				# initial_turbo disabled due to: https://github.com/Fourdee/DietPi/issues/1836
+				#"ARM Initial Turbo"*)
 
-					MIN_VALUE=0
-					MAX_VALUE=60
-					G_WHIP_DEFAULT_ITEM=$current_initial_turbo
-					G_WHIP_INPUTBOX "Activates turbo mode during boot ($frequency_max_cpu_governor MHz), for the duration of value in seconds. \n - Recommended value is 20 \n - Valid range $MIN_VALUE (disabled) - $MAX_VALUE"
-					if (( $? == 0 )); then
+				#	MIN_VALUE=0
+				#	MAX_VALUE=60
+				#	G_WHIP_DEFAULT_ITEM=$current_initial_turbo
+				#	G_WHIP_INPUTBOX "Activates turbo mode during boot ($frequency_max_cpu_governor MHz), for the duration of value in seconds. \n - Recommended value is 20 \n - Valid range $MIN_VALUE (disabled) - $MAX_VALUE"
+				#	if (( $? == 0 )); then
 
-						if G_CHECK_VALIDINT $G_WHIP_RETURNED_VALUE &&
-							(( $G_WHIP_RETURNED_VALUE <= $MAX_VALUE && $G_WHIP_RETURNED_VALUE >= $MIN_VALUE )); then
+				#		if G_CHECK_VALIDINT $G_WHIP_RETURNED_VALUE &&
+				#			(( $G_WHIP_RETURNED_VALUE <= $MAX_VALUE && $G_WHIP_RETURNED_VALUE >= $MIN_VALUE )); then
 
-							sed -i "/initial_turbo=/c\initial_turbo=$G_WHIP_RETURNED_VALUE" /DietPi/config.txt
-							REBOOT_REQUIRED=1
+				#			G_CONFIG_INJECT 'initial_turbo=' "initial_turbo=$G_WHIP_RETURNED_VALUE" /DietPi/dietpi.txt
```

Comment:
Should be:
```
config.txt
```

Reply:
Although, disabled. May as well just remove option from code, we can always use git history to re-add if required?

Reply:
@Fourdee 
Yeah good idea. The file is long enough 😋. Maybe we can open an issue about initial_turbo, to not forget and by times re-check with current RPi firmware and re-add setting in case?